src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageAction.java

```diff
@@ -80,31 +80,38 @@ private void addFeedbackSubmissionStatusMessageIfNotRedirected() {
     }
 
     /**
-     * Returns if the page is redirected from other pages that are mapped to subclasses of
+     * Returns if the page is redirected from either pages that are mapped to subclasses of
```

Comment:
Sorry this was my typo, can you revert it to `other`?

Reply:
Extremely sorry for not checking it. This should have caught my eyes.